### DISCUSSION TITLE
Encryption

### DISCUSSION THREAD

**Discussion by WooAf** [discord-tickets/bot#260] [Support (Q&A)]
What would be the right way to decrypt the database using PHP ?

**Comment by eartharoid** [comment]
The bot uses [cryptr](https://github.com/MauriceButler/cryptr) so you need to find how to decrypt an `aes-256-gcm`-encrypted string in PHP. You'd also need to find a way of getting the salt (look at [how cryptr does it](https://github.com/MauriceButler/cryptr/blob/master/index.js)).

**Reply by WooAf** [reply]
> The bot uses [cryptr](https://github.com/MauriceButler/cryptr) so you need to find how to decrypt an `aes-256-gcm`-encrypted string in PHP. You'd also need to find a way of getting the salt (look at [how cryptr does it](https://github.com/MauriceButler/cryptr/blob/master/index.js)).

Yeah, I was wondering if someone has a PHP function already coded
